Dockerfile

```diff
@@ -42,10 +40,14 @@ COPY --from=build /splinterdb-install/lib/ /usr/local/lib/
 COPY --from=build /splinterdb-install/include/splinterdb/ /usr/local/include/splinterdb/
 
 # Copy over the test binaries under bin/ (recursively) and the test script
-COPY --from=build /splinterdb-src/bin/ /splinterdb/bin/
+# (Default BUILD_DIR is 'build'.)
+COPY --from=build /splinterdb-src/build/bin/ /splinterdb/build/bin/
```

Comment:
This is needed because, by default, the build outputs come to `build/bin`. W/o this fix, one of the CI-jobs that runs thru Docker image building fails to run any tests.